.github/workflows/compilation_on_android_ubuntu.yml

```diff
@@ -527,7 +527,7 @@ jobs:
         working-directory: ./core/iwasm/libraries/lib-socket/test/
 
       - name: run tests
-        timeout-minutes: 20
```

Comment:
How long does it take to run the stress test? If that all adds up to 60 minutes now, I think it's reasonable to keep for nightly only.

Reply:
it's a bit tricky 

actually it should be 2 times less. This constraint applies because `jit` is basically 2 bundle runs 

here's the ticket to set timeouts based on the platform. So, all platforms <= 30 minutes, jit <= 60 minutes

Reply:
Even if

Reply:
Seems it took long time to run the stress cases:
https://github.com/bytecodealliance/wasm-micro-runtime/actions/runs/5761023933/job/15618324568
https://github.com/bytecodealliance/wasm-micro-runtime/actions/runs/5761023933/job/15618325167
https://github.com/bytecodealliance/wasm-micro-runtime/actions/runs/5761023933/job/15618325916
https://github.com/bytecodealliance/wasm-micro-runtime/actions/runs/5761023933/job/15618326547
https://github.com/bytecodealliance/wasm-micro-runtime/actions/runs/5761023933/job/15618326990

Almost close to 30 min for each running mode, but it is only 4-5 min now:
https://github.com/bytecodealliance/wasm-micro-runtime/actions/runs/5791716284/job/15697007728
https://github.com/bytecodealliance/wasm-micro-runtime/actions/runs/5791716284/job/15697008239
https://github.com/bytecodealliance/wasm-micro-runtime/actions/runs/5791716284/job/15697009623

Shall we put the stress test in nightly run?

Reply:
we can either decrease the constants (what should be harmless, because if it did 100'000 threads it's really likely to do 10 times more without errors or put to nightly

Reply:
How about decreasing the constants to ensure it doesn't increase too much time in this PR, and then adding another PR for stress test?
We had better not make it take too long to run the CIs of a PR, developer may be eager to see the running result.

Reply:
I've changed the constants. I will wait for execution and will adjust timeouts respectively

Reply:
OK, the nightly_run CI reported failure:

https://github.com/bytecodealliance/wasm-micro-runtime/actions/runs/5801072533/job/15724690363#step:12:488

```
stress_test_threads_creation.wasm is a stress test
Assertion failed: status_code == 0 && "Thread creation should succeed" (stress_test_threads_creation.c: spawn_thread: 55)
Exception: unreachable
Stress test stress_test_threads_creation.wasm FAILED with code  1
```

Reply:
for some reason `multitier-jit` looks really problematic in terms of threads creation. 

I'm not sure whether it's by design or it can be a potential sign of some bug